client-app/core/composables/useMutationBatcher.ts

```diff
@@ -104,17 +133,38 @@ export function useMutationBatcher<TData, TVariables extends object>(
 
   async function executeBatch(): Promise<FetchResult<TData> | null> {
     abortController = new AbortController();
-    return await mutation(batch, {
+    return await mutation(batch.value, {
       context: { fetchOptions: { signal: abortController.signal } },
+      ...mutationOptions,
     });
   }
 
-  function resetBatchState() {
+  function resetBatchState(resetCount = true) {
     overflowed.value = false;
     abortController = null;
-    batch = {} as TVariables;
-    calledCount = 0;
+    batch.value = {} as TVariables;
+    loading.value = false;
+    if (resetCount) {
```

Comment:
we don't need to reset count when explicitly call abort method (for batchSynchronizer service for example). I refactor it a bit to avoid that condition, it should be better.